### DISCUSSION TITLE
error on SBS

### DISCUSSION THREAD

**Discussion by Vinsamlegast78** [section83/MacYTDL#61] [General]
Hi and thank you for this 
I am on Mac 12.6.1 and MacYtDL  Version 1.21.2 - this is no longer working for SBS (ABC still works
When it gets to the screen to select an episode it fails and none of the checkboxes appear.
See the screenshots (ABC is working) Any ideas of what the issue is?

<img width="312" alt="Screen Shot 2022-11-03 at 6 23 18 pm" src="https://user-images.githubusercontent.com/117354041/199697357-ec426abf-c97d-49ed-b1ac-0dad877b5548.png">
<img width="469" alt="Screen Shot 2022-11-03 at 6 23 04 pm" src="https://user-images.githubusercontent.com/117354041/199697386-21620df7-6bb4-4ce5-8340-6450926e7f62.png">

**Comment by section83** [comment]
Now, that is ugly !  My apologies. SBS changed how they serve videos (which is their right) which broke the code in MacYTDL that looks for episodes of shows. I will work on a fix over the weekend.

In the meantime, you can still download individual videos using this procedure:

- In your web browser, navigate to the relevant "show" page e.g. https://www.sbs.com.au/ondemand/tv-series/war-of-the-worlds.
- Start playing an episode.
- Copy the URL in the address bar e.g. Season 1, Episode 1 is https://www.sbs.com.au/ondemand/watch/1756135491797.
- Open MacYTDL.
- Paste that episode URL into the "Paste URL here" box.
- Click on "Continue".

That method should work for most videos. Movies should work with URLs like this: https://www.sbs.com.au/ondemand/movie/gone-girl/2068778563699.

Many thanks for posting about the issue.

**Reply by Vinsamlegast78** [reply]
Thanks so for much for you appreciated speedy response - this is now working for me (so much to watch and not enough time!)

**Comment by Vinsamlegast78** [comment]
Thank you for the latest update - it's much easier now - much appreciated.

**Comment by Vinsamlegast78** [comment]
HI - I downloaded your latest version - but had to go back to the previous version due to errors for season 2 on sbs

**Comment by Vinsamlegast78** [comment]
Downloading the 2nd series with your previous advice - so thank you this is still worling

**Comment by section83** [comment]
> errors for season 2 on sbs

MacYTDL still cannot find all seasons of an SBS show - only the front-most season.  However, all links to videos should work (e.g. https://www.sbs.com.au/ondemand/tv-series/cardinal/season-2/cardinal-s2-ep2/1222196803960 and https://www.sbs.com.au/ondemand/watch/1222199363506).

Can you take a capture of the error ? Which show were you trying ? What was the URL ?

Thanks.

**Comment by Vinsamlegast78** [comment]
trying Season 2 of Spring Tide https://www.sbs.com.au/ondemand/tv-series/spring-tide/season-2/spring-tide-s2-ep1/1868699203557 gives this error

Error in reading log file: Macintosh HD:Users:myname:Library:Preferences:MacYTDL:youtube-dl_log-WARNING/ [generic] Falling back on generic information extractor.
ERROR/ [generic] Unsupported URL/ https/:www.sbs.com-Thursday_2_February_2023_at_3_50_16_pm.txt
The error reported was Can’t make file (file "Macintosh HD:Users:myname:Library:Preferences:MacYTDL:youtube-dl_log-WARNING/ [generic] Falling back on generic information extractor.
ERROR/ [generic] Unsupported URL/ https/:www.sbs.com-Thursday_2_February_2023_at_3_50_16_pm.txt") into type file.  Whereas this url will work: https://www.sbs.com.au/ondemand/tv-series/spring-tide/season-2/spring-tide-s2-ep2/1868699203561

**Comment by section83** [comment]
[Vinsamlegast78](https://github.com/Vinsamlegast78), many thanks. It will take a day or two to work out what's going on. I'll reply here when I have news.

UPDATE: I'm not able to replicate the problem. Both URLs work for me. Can you check you are using v1.23 of MacYTDL ? That version had fixes which affected downloading from SBS.

There might be a conflict in the MacYTDL Settings. Can you email to me a copy of your MacYTDL settings ? The settings are stored here:

* [Mac HD]/Users/[Your Home]/Library/Preferences/MacYTDL/MacYTDL.plist

My email address is in the MacYTDL app > Utilities > "About MacYTDL" box.

Many thanks.

**Comment by Vinsamlegast78** [comment]
Many thanks for this. Yes I am on v1.23.  I've sent you an email with a screenshot and the .plist

Cheers

**Comment by Vinsamlegast78** [comment]
Hi Getting errrors on SBS again

There was an error with the URL you entered:

'https://www.sbs.com.au/ondemand/watch/1746589251656'

The error message was: 

WARNING: [ThePlatform] Failed to download m3u8 information: HTTP Error 400: Bad Request
ERROR: [ThePlatform] gte8kZbyee1x: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; type  yt-dlp -U  to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.

Quit, OK to return or Download to try anyway.

**Comment by section83** [comment]
Thanks for making contact.

I tried that URL this morning and found it works normally.  Make sure you are using the latest version of YT-DLP – it should be version 2023.10.13.  To update your copy of YT-DLP, go to "Utilities" in MacYTDL and choose to update the stable build then click on "Start".

Cheers.

**Comment by Vinsamlegast78** [comment]
Thanks so much for your speedy response - it's working again! In future, I will make sure I update the latest version of YL-DLP

**Comment by section83** [comment]
No worries.  Stable releases YT-DLP are less frequent now so, you can choose to use the latest nightly build. I find nightly builds useful when I know the YT-DLP developers have fixed a particular issue but not put out a stable release.

Cheers.
